.github/CHANGELOG.md

```diff
@@ -85,6 +85,11 @@
 
 <h3>Bug fixes</h3>
 
+* Fixes a bug in the unit tests ``test_single_ex_unitary_operations`` and
+  ``test_double_ex_unitary_operations`` in order to test the correctness of the
+  entire set of quantum gates applied for building the UCCSD ansatz.
+  [(#659)](https://github.com/XanaduAI/pennylane/pull/659)
```

Comment:
@agran2018 I wouldn't worry about putting this in the changelog. The changelog usually specifies the changes from the previous release --- since the `DoubleExcitationUnitary` and `SingleExcitationUnitary` templates are new, this bug didn't exist 🙂 

Instead, you can link this PR (#659) under the changelog entries for the Double and Single excitation templates:

```markdown
* Contains the new template ``DoubleExcitationUnitary`` implementing the quantum circuit to
  exponentiate the Coupled-Cluster double excitation operator. This template is required to
  build the Unitary Coupled-Cluster Singles and Doubles (UCCSD) ansatz for VQE simulations.
  [(#638)](https://github.com/XanaduAI/pennylane/pull/638)
  [(#659)](https://github.com/XanaduAI/pennylane/pull/659)

* Contains the new template ``SingleExcitationUnitary`` implementing the quantum circuit to
  exponentiate the Coupled-Cluster single excitation operator. This template is required to
  build the Unitary Coupled-Cluster Singles and Doubles (UCCSD) ansatz for VQE simulations.
  [(#622)](https://github.com/XanaduAI/pennylane/pull/622)
  [(#659)](https://github.com/XanaduAI/pennylane/pull/659)
```

Reply:
Thanks @josh146!, understood.

Reply:
done